elliottlib/cli/verify_attached_bugs_cli.py

```diff
@@ -213,8 +214,8 @@ async def _verify_attached_flaws_for(self, advisory_id: int, attached_trackers:
                            "You need to drop those flaw bugs or attach corresponding tracker bugs.")
 
         # Check if advisory is of the expected type
-        advisory_info = await self.errata_api.get_advisory(advisory_id)
```

Comment:
get advisory can use Erratum

Reply:
so no need to decode json and Erratum is very reliable, and after this only cve_package_exclusions API which not implement in Erratum will called by request